### DISCUSSION TITLE
Possibility to disable "Edit Raw Value"

### DISCUSSION THREAD

**Discussion by mkduew** [directus/directus#23009] [Feature Requests]
### Summary

it should be possible to disable that feature complete (as a access setting) or per field individually. 

### Motivation

This can create huge problems if inexperienced users (data editors) use that feature. Especially if you work with drop down list fields.

### Detailed Design

---

### Requirements List

---

### Drawbacks

---

### Alternatives

---

### Adoption Strategy

---

**Comment by mkduew** [comment]
already discussed here, but closed: https://github.com/directus/directus/discussions/16378 - I cannot understand why such an important feature request was ignored.

**Reply by rijkvanzanten** [reply]
> I cannot understand why such an important feature request was ignored.

Because we prioritize feature requests by community engagement and upvotes, as [per my comment](https://github.com/directus/directus/discussions/16378#discussioncomment-5866129) on there 👍🏻

**Reply by alejandrolga** [reply]
I am also interested in individually disabling the edit raw value options, it does not help me to disable everything because in some cases if I use it, even if it can be managed by roles it would be perfect.

**Reply by yakupsogut** [reply]
> > I cannot understand why such an important feature request was ignored.
> 
> Because we prioritize feature requests by community engagement and upvotes, as [per my comment](https://github.com/directus/directus/discussions/16378#discussioncomment-5866129) on there 👍🏻

Hello, I have developed this feature and am ready to contribute. If you could assign me to the relevant branch, I can push commits directly and open a pull request. Thank you for your consideration!

**Comment by drkirkham** [comment]
This seems to be an important feature that should be added - at the minute it undermines the appeal of Directus.

**Comment by yakupsogut** [comment]
Hello, I have developed this feature and am ready to contribute. If you could assign me to the relevant branch, I can push commits directly and open a pull request. Thank you for your consideration!

**Reply by rijkvanzanten** [reply]
How would you implement this? Is it a setting on the field or a permissions or..?

**Comment by Abdallah-Awwad** [comment]
One of the problems I faced because of this, when I used the slugify in the interface tab, one of my customers was using the edit raw value to ignore the slug behavior 🤦🏻‍♂️
(had to use extra validation to keep it restricted)
